src/openforms/registrations/contrib/objects_api/templatetags/registrations/contrib/objects_api/json_tags.py

```diff
@@ -11,17 +13,20 @@
 
 
 @register.simple_tag(takes_context=True)
-def uploaded_attachment_urls(context):
+def uploaded_attachment_urls(context: template.Context) -> SafeString:
     """
     Output a sequence of attachment URLs as a JSON-serialized list.
     """
-    # attachments is a list of URLs
+    warnings.warn(
+        "'uploaded_attachment_urls' is deprecated, use 'as_json' with a path argument instead.",
+        DeprecationWarning,
+    )
```

Comment:
Before this PR, `uploaded_attachmen_urls` tried to get attachment URLs in a safe way (`context.get("submission", {}).get("uploaded_attachment_urls", [])`).

This is not the case now with `as_json`? `as_json submission.uploaded_attachment_urls` will fail if `submission` isn't a `dict`.

Reply:
Template tags are allowed to crash, template filters aren't, so accepted risk. We deliberately avoid exposing model instances in the user-controlled template evaluation since we don't want them to access the DB/call methods like that.